pkg/controller/azvolume.go

```diff
@@ -604,7 +587,7 @@ func createAzVolumeFromPv(ctx context.Context, pv v1.PersistentVolume, azVolumeC
 			volumeParams = pv.Spec.CSI.VolumeAttributes
 		}
 
-		azvolume := diskv1alpha2.AzVolume{
+		azVolume := diskv1alpha2.AzVolume{
```

Comment:
Are we planning to change the maxMountReplicaCount value for generic ephemeral volumes in this PR?

Reply:
I think we will be addressing that with a separate PR.